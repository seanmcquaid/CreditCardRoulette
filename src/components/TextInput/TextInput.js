import React from "react";
import styled from "styled-components/native";
import { TextInput } from "react-native";

export default styled(TextInput)`
    border : 2px solid black;
    padding : 10px;
    margin : 10px;
    width : 180px;
    background-color : white;
`;