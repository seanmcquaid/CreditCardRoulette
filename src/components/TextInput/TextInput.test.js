import React from "react";
import {render} from "react-native-testing-library";
import TextInput from "./TextInput";

describe("<TextInput/>", () => {
    it("Snapshot matches correctly", () => {
        const textInput = render(<TextInput/>);
        expect(textInput).toMatchSnapshot();
    });
});