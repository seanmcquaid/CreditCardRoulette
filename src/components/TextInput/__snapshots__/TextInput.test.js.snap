// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextInput/> Snapshot matches correctly 1`] = `
<TextInput
  allowFontScaling={true}
  rejectResponderTermination={true}
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderColor": "black",
        "borderStyle": "solid",
        "borderWidth": 2,
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 10,
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 10,
        "width": 180,
      },
    ]
  }
  underlineColorAndroid="transparent"
/>
`;
